### DISCUSSION TITLE
How should we name the logging / metrics collection of TDP?

### DISCUSSION THREAD

**Discussion by leopaul36** [TOSIT-IO/TDP#63] [Polls]
[Google Cloud's definitions](https://cloud.google.com/architecture/devops/devops-measurement-monitoring-and-observability):

> **Monitoring** is tooling or a technical solution that allows teams to watch and understand the state of their systems. Monitoring is based on gathering predefined sets of metrics or logs.

> **Observability** is tooling or a technical solution that allows teams to actively debug their system. Observability is based on exploring properties and patterns not defined in advance.

**Comment by rpignolet** [comment]
Repo URL is https://github.com/TOSIT-IO/tdp-observability.
